tst/util_test.py

```diff
@@ -35,6 +43,19 @@
 )
 
 
+class TestPredictor(SklearnPredictor):
```

Comment:
Do you have an example somewhere, which also outputs predictive variances?

Reply:
The example with a proper model is in launch examples. I wanted to keep this as simple as possible but updated the stds to be non-zero to make sure it does not crash anything